litellm/llms/ollama.py

```diff
@@ -121,6 +124,61 @@ def get_config(cls):
         }
 
 
+class ResponseJSON(TypedDict, total=False):
```

Comment:
I found myself frequently having to revisit the documentation to check the response shape so I added these `TypedDict` specs to make the Ollama API specification a bit clearer.